express/app/services/userService.js

```diff
@@ -0,0 +1,10 @@
+const user = require('../models').user;
+
+exports.create = async userFields =>
+  user.findOne({ where: { email: userFields.email } }).then(userInstance => {
+    if (userInstance) {
+      throw new Error('Email already exists');
+    } else {
+      return user.create(userFields);
+    }
```

Comment:
Service doesn't have the responsibility of this. You must check if user exists in the controller.